src/Components/BoardGame/BoardGame.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export const BoardGame = (props) => {
+  return (
+    <main>
+      <h1 className='name'>{props.name}</h1>
+      <h2 className='description'>{props.description}</h2>
+      <p className='player-count'>Players: {props.min_players} - {props.max_players}</p>
+      <p className='average-time'>Estimated Time: {props.average_time} minutes</p>
+      <img className='image' src={props.image} alt={`poster for ${props.name}`}></img>
+    </main>
```

Comment:
The components all look good here, I know we talked about adding a `choose` or `skip` button so we may have to add that in here as well.

Reply:
@BlakeDonn  Are you referring to the buttons that should be shown when the suggested activity is rendered? I think I imagined this file in particular as what should be rendered if the user chooses to see the details for the suggested activity.  Right?

Reply:
Oh you are totally right, I forgot about the step before this. Would that be a conditional render on this component?

Reply:
I think we were considering adding a `Result` component to render that suggested activity, but we could certainly do some conditional rendering within the `Form`.  Up to us!

Reply:
Honestly a `Result` component sounds cleaner to me, since we will be doing the logic for the possible activities in app we can just pass all the possibilities to the `Result` component and have access to all the possibilities.

Reply:
I think so, too.

Reply:
I think that given our time frame we should go with displaying the result component.